docs/source/get_started/install.md

```diff
@@ -80,7 +80,11 @@ To exit `kedro-environment`:
 ```bash
 conda deactivate
 ```
+### Optional: Setup Kedro with IDE
```

Comment:
```suggestion
### Optional: Add Kedro support to your IDE
```

? (Unusure about this one as well, requesting @stichbury's opinion)

Reply:
> [Integrate ESLint in VSCode with the official extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)

Integrate Kedro in VS Code with the official extension? @astrojuanlu

Reply:
+1!